components/gly_gen/glycosight/__init__.py

```diff
@@ -0,0 +1,2 @@
+def load_mzid_file(file_details):
+    return file_details
```

Comment:
This isn't doing anything yet?

Reply:
This just passes the `File` object up for processing by downstream metanodes.

Reply:
We can keep this in the codebase for now, as an example if we do end up doing server-side work in python.